helm/charts/oathkeeper/templates/deployment-controller.yaml

```diff
@@ -42,16 +42,21 @@ spec:
         - name: {{ include "oathkeeper.name" . }}-config-volume
           configMap:
             {{- if .Values.demo }}
-            name: {{ include "oathkeeper.fullname" . }}-config-demo
+            name: {{ include "oathkeeper.name" . }}-config-demo
```

Comment:
Please do not change this, and use `fullname`. This allows installation of multiple instances without resource overlapping.